src/LaunchDarkly.Client/LaunchDarkly.Client.nuspec

```diff
@@ -14,5 +14,8 @@
     <releaseNotes>Switch to asynchronous HTTP client</releaseNotes>
     <copyright>$copyright$</copyright>
     <tags>Feature Configuration</tags>
+     <dependencies>
+      <dependency id="Newtonsoft.Json" version="8.0.3" />
```

Comment:
This is the change that seems to have definitively fixed the problem our customer saw and that I reproduced locally.